helm-charts/basehub/values.yaml

```diff
@@ -405,7 +405,8 @@ jupyterhub:
                     description: An IDE For R, created by the RStudio company
     initContainers:
       - name: templates-clone
-        image: alpine/git
+        image: alpine/git:2.40.1
+        imagePullPolicy: IfNotPresent
```

Comment:
I think using a tag besides `latest`, or to use `latest` and specify `IfNotPresent`, is sufficient to ensure the behavior of `IfNotPresent` I think.

In a way I like that we use `:latest` here rather than having another image that can get outdated or that we automatically merge automated version bumps without further review because we assume its fine no matter what?

To pin:
- A minus: we ~need automation to bump it over time

To not pin:
- A minus: we don't know for sure when something updates, because `:latest` combined with `IfNotPresent` makes us not know that

In this case, I think I'm leaning towards favoring not needing the complexity of updating this automatically or the alternative drawback of possibly exposing security vulernabilities that we don't patch.

> - If the image tag is :latest, the imagePullPolicy will be automatically set to Always. 
> - If the image tag isn’t :latest, the imagePullPolicy will be automatically set to IfNotPresent. 
And if you don’t set any image tag, the imagePullPolicy will be automatically set to.

Reply:
We can go for this as it is though, I mostly wanted to ensure that that we don't spread an understanding that we would need IfNotPresent specified alongside any image tag beisdes `:latest`